Packs/Jira/Scripts/JiraCreateIssueExample/JiraCreateIssueExample_test.py

```diff
@@ -1,82 +1,134 @@
-from JiraCreateIssueExample import add_additional_args, get_known_args_from_input, get_additional_args_from_input, DEFAULT_ARGS
+from JiraCreateIssueExample import DATE_FORMAT, validate_date_field, parse_custom_fields, add_custom_fields,\
+    rm_custom_field_from_args
 import pytest
 
 
-@pytest.mark.parametrize("input,expected", [
-    ({DEFAULT_ARGS[0]: "system down"}.items(), {"summary": "system down"}),
-    ({DEFAULT_ARGS[0]: "system down", "unknown_key": "some key"}.items(), {"summary": "system down"}),
-    ({"unknown_key": "some key"}.items(), {})
+@pytest.mark.parametrize("due_date", [
+    ("2022-01-01"),
+    ("2022-12-12T13:00:00")
 ])
-def test_get_known_args_from_input(input, expected):
-
+def test_validate_date_field_data_remains(due_date):
     """
     Given:
-        - A dictionary view with Jira fields
+        - A date.
+
     When:
-        - 1 known argument key is supplied
-        - 1 known argument key is supplied, 1 unknown argument key is supplied
-        - 0 known arguments are supplied,  1 unknown argument key is supplied
+        - Case A: The date is in valid format.
+        - Case B: The date has a time included
 
     Then:
-        - dict with known argument key is returned
-        - dict with known argument key is returned
-        - empty dict
+        - Case A: No exception is thrown.
+        - Case B: A `ValueError` exception is thrown.
     """
 
-    args = get_known_args_from_input(input)
+    try:
+        validate_date_field(due_date)
+    except ValueError as ve:
+        assert "unconverted data remains" in str(ve)
+
 
-    assert isinstance(args, dict)
-    assert args == expected
+@pytest.mark.parametrize("due_date", [
+    ("2022-1-1"),
+    ("2022-31-31")
+])
+def test_validate_date_field_format(due_date):
```

Comment:
Looking at this test after reviewing the previous one, it is better that you will create one function for success cases and one for failures such that each failure will also have a different Exception to catch, like this:
```python
def my_test(due_date, exception_type) 
    with pytest.raises(exception_type) as e:
        validate_date_field(due_date)
```

Reply:
Addressed in [`2b875a6` (#22601)](https://github.com/demisto/content/pull/22601/commits/2b875a6111d34fc492e18ac7ee5335e81c46a278).